ui/app/AppLayouts/Onboarding/views/SeedPhraseInputView.qml

```diff
@@ -181,46 +182,50 @@ Item {
                 height: (grid.cellHeight - 8)
                 Behavior on width { NumberAnimation { duration: 180 } }
                 textEdit.text: {
-                    let pos = parseInt(seedWordInput.leftComponentText)
-                    for (var i in root.mnemonicInput) {
-                        let p = root.mnemonicInput[i]
+                    const pos = seedWordInput.mnemonicIndex
+                    for (let i in root.mnemonicInput) {
+                        const p = root.mnemonicInput[i]
                         if (p.pos === pos) {
                             return p.seed
                         }
                     }
                     return ""
                 }
-                leftComponentText: grid.wordIndex[(grid.count/6)-2][index]
-                inputList: BIP39_en { }
+
+                readonly property int mnemonicIndex: grid.wordIndex[(grid.count / 6) - 2][index]
+
+                leftComponentText: mnemonicIndex
```

Comment:
is this still needed?

Reply:
yes, need to display left-side labels with an ordinal number

Reply:
can't we use mnemonicIndex right away?